config/routes.rb

```diff
@@ -228,9 +228,11 @@ def healthcare_routes
       end
       resources :roles, only: [:index]
     end
+    resources :translation_keys
```

Comment:
I think we can significantly limit the routes here, maybe only: [:index, :update]?